components/controls/__tests__/ActionsDropdown-test.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Sonar UI Common
+ * Copyright (C) 2019-2019 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+/* eslint-disable sonarjs/no-duplicate-string */
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import { click } from '../../../helpers/testUtils';
+import { PopupPlacement } from '../../ui/popups';
+import ActionsDropdown, { ActionsDropdownDivider, ActionsDropdownItem } from '../ActionsDropdown';
+
+describe('ActionsDropdown', () => {
+  it('should render correctly', () => {
+    expect(
+      shallow(
+        <ActionsDropdown
+          // eslint-disable-next-line react/no-children-prop
```

Comment:
Is it raising blocking issues ? Quality gate wise ? If not maybe it's better to just have them reported on SonarQube instead of polluting the code with those comments ?

Reply:
Better yet, pass the children as actual children!

Reply:
Ah yeah indeed xD

Reply:
Yeah, leftover from a quick refactor. It was part of a `shallowRender()` call, and I wanted to let the children be passed as props. But not relevant here. Will change.